Fixes warning:
main.c: In function 'load_song':
main.c:56:23: warning: incompatible implicit declaration of built-in function 'strlen'
diff --git a/main.c b/main.c
index ae852a7..417ff62 100644
--- a/main.c
+++ b/main.c
@@ -21,6 +21,7 @@
  * 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#include <string.h>
 #include <unistd.h>
 #include "timers.h"
 #include "bb.h"
